daemon/src/projection.rs

```diff
@@ -362,7 +362,15 @@ impl Cfd {
             CetTimelockConfirmedPostOracleAttestation { .. } => {
                 (CfdState::PendingCet, self.actions)
             }
-            RefundTimelockConfirmed { .. } => (self.state, self.actions),
+            RefundTimelockExpired { .. } => {
+                // TODO: Not sure what to display here! There are
```

Comment:
👀 I'm unsure about this change.

Reply:
I think it is fine to prioritize displaying the refund state here temporarily. If we end up in a race condition on which TX makes it. Whatever gets confirmed will fix it later.